### PR TITLE
Add notifications

### PR DESCRIPTION
**Pull request by MaksimK193**

### CODE REVIEW

File: SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    
+    var body: some View {
+        VStack {
+            DatePicker("", selection: $notificationDate, in: Date.now...)
+                .labelsHidden()
+            TextField("Notification text", text: $notificationText)
+                .textFieldStyle(.roundedBorder)
+            Button("Schedule notification") {
+                let center = UNUserNotificationCenter.current()
+                center.requestAuthorization(options: [.alert, .badge, .sound]) { success, error in
```
Comment (IslamRustamov98) [inline SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift:22]: is it possible to move this function out of var body?

File: SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                    if success {
+                        print("All set!")
```
Comment (IslamRustamov98) [inline SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift:24]: no prints

File: SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                        content.title = "Schedule notification"
+                        content.subtitle = notificationText
+                        content.sound = UNNotificationSound.default
+                        
+                        let comps = Calendar.current.dateComponents([.year, .month, .day, .hour, .minute], from: notificationDate)
+                        let trigger = UNCalendarNotificationTrigger(dateMatching: comps, repeats: false)
+                        let request = UNNotificationRequest(identifier: UUID().uuidString, content: content, trigger: trigger)
+                        center.add(request)
+                    } else if let error = error {
+                        print(error.localizedDescription)
```
Comment (IslamRustamov98) [inline SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift:35]: this print is ok

File: SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift (hunk 2 above)
Comment (MaksimK193) [reply SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift]: fixed

File: SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift (hunk 1 above)
Comment (MaksimK193) [reply SwiftUIStudy/Flows/ScheduleNotification/ScheduleNotification.swift:22]: fixed

**Pull request merged**